src/workloads/issues/ConnectionsBuildup.yml

```diff
@@ -1,8 +1,10 @@
 SchemaVersion: 2018-07-01
 Owner: "@mongodb/product-perf"
 Description: |
-  This workload created to reproduce SERVER-53853: Large buildup of mongos to mongod connections and
-  low performance with secondaryPreferred reads
+  This workload was created to reproduce SERVER-53853: Large buildup of mongos to mongod connections and
+  low performance with secondaryPreferred reads. For running both with and without sharding enabled.
```

Comment:
The 'For running both with and without sharding enabled.' reads as if it was part of a longer sentence that got edited.

Maybe we can change it to something like the following:

```
This workload was originally two separate files, ConnectionsBuildup.yml and ConnectionsBuildupNoSharding.yml. It was merged into a single file when "OnlyRunInInstance: sharded"
became an option.
```

Reply:
Done